infra/launch.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+import argparse
+import getpass
+import subprocess
+
+from infra import deploy
+from google.cloud import storage
+
+
+# Oddly enough, there's no API to simply fetch the current gcloud configuration...
+def gcloud_config():
+    client = storage.Client()
+    return {
+        "project": client.project,
+    }
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--tpu", type=str, required=True)
+    parser.add_argument("--project", type=str, default=gcloud_config()["project"])
+    parser.add_argument("--zone", type=str, required=True)
+    parser.add_argument("--docker_repository", type=str, required=True)
+    parser.add_argument("--wandb_api_key", type=str, required=False)
+    parser.add_argument("--hf_token", type=str, required=False)
+    parser.add_argument("--image_name", type=str, required=False)
+    parser.add_argument("command", nargs=argparse.REMAINDER)
+
+    args = parser.parse_args()
+
+    zone = args.zone
+    region = "-".join(zone.split("-")[:-1])
+    project = args.project
+    tpu_name = args.tpu
+    wandb_api_key = args.wandb_api_key
+    hf_token = args.hf_token
+    image_id = args.image_name or f"levanter-{getpass.getuser()}"
+    docker_repository = args.docker_repository
+    command = args.command
+
+    if command[0] == "--":
+        command = command[1:]
+
+    print(f"Launching {image_id} on {tpu_name}...")
+    deploy.build_and_push_docker_image(
+        project_id=project,
+        region=region,
+        repository=docker_repository,
+        image_name=image_id,
+    )
+
+    full_image_id = f"{region}-docker.pkg.dev/{project}/{docker_repository}/{image_id}"
+
+    docker_command = [
+        "docker",
+        "run",
+        "-t",
+        "--privileged",
+        "--shm-size=32gb",
+        "--net=host",
+        "-v",
+        "/home:/home",
+        "-v",
+        "/tmp:/tmp",
+        "-e",
+        "PYTHONPATH=/opt/levanter:/opt/levanter/src:/opt/levanter/examples",
```

Comment:
```suggestion
        "PYTHONPATH=/opt/levanter:/opt/levanter/src:/opt/levanter/examples:/opt/levanter/tests",
```

(I've started running tests on TPU in CI)

Reply:
Ack, applied.